atom/browser/native_window_mac.mm

```diff
@@ -397,6 +397,11 @@ - (NSTouchBarItem*)touchBar:(NSTouchBar*)touchBar
     return nil;
 }
 
+-(void)setEscapeTouchBarItem:(mate::PersistentDictionary)item {
+  if (self.touchBar && atom_touch_bar_)
```

Comment:
This should be `if (atom_touch_bar_ && self.touchBar)` since the object may not respond to `self.touchBar` and would crash if called on a non-touchbar OS version.